.github/container/Dockerfile.base

```diff
@@ -203,12 +199,5 @@ RUN ln -s /opt/jax_nsys/nsys-jax-ensure-protobuf /usr/local/bin/
 ## Copy manifest file to the container
 ###############################################################################
 
-# Set the manifest env vars
-ENV MANIFEST_FILE=${DEST_MANIFEST_DIR}/${SRC_MANIFEST_FILE}
-# Copy all required files for manifestation
-ADD ${SRC_MANIFEST_FILE} \
-    bump.sh \
-    bump-openxla-triton.sh \
-    create-distribution.sh \
-    ${DEST_MANIFEST_DIR}/
-COPY patches/ ${DEST_MANIFEST_DIR}/patches/
+ADD manifest.yaml /opt/manifest.d/
+COPY patches/ /opt/manifest.d/patches/
```

Comment:
Also we need sort of flag whether to apply patch or not to a custom source. Let's say, you wanna build `flax` from custom repo, but you don't want to apply our patches.

Reply:
The patches are only applied when building the Rosetta container. Maybe @terrykong can answer if this is an immediate need or not.

Reply:
@terrykong If we apply patches only for Rosetta, maybe, let's move it to rosetta docker files? WDYT?

Reply:
Moving rosetta-specific stuff out of the base container sounds reasonable to me, but I don't think it's directly linked to this PR so I'm marking this resolved.